src/sdam/monitor.rs

```diff
@@ -607,17 +593,28 @@ impl MonitorRequestReceiver {
         err
     }
 
-    /// Wait for a request to immediately check the server to come in, guarded by the provided
-    /// timeout. If a cancellation request is received indicating the topology has closed, this
-    /// method will return. All other cancellation requests will be ignored.
-    async fn wait_for_check_request(&mut self, timeout: Duration) {
+    /// Wait for a request to immediately check the server to be received, guarded by the provided
+    /// timeout. If the server associated with this monitor is removed from the topology, this
+    /// method will return.
+    ///
+    /// The `delay` parameter indicates how long this method should wait before listing to requests.
+    /// It is covered by the provided timeout.
```

Comment:
does "covered by" mean "counts toward"? like if `delay` is 1s and timeout is 5s, the method delays for 1s then times out 4s if no requests come in? (I think yes from reading the logic)

Reply:
Yep, updated the docstring to more be clearer about this.